app/styles/app.css

```
@@ -1,6 +1,8 @@
 @import url("./radix/switch.css");
+@import url("./components/button.css");
+@import url("./components/link.css");
+@import url("./radix/navigation-menu.css");
```

ordenemos los imports por tipo, asi a simple vista podemos ver cuantos radix components ya trabajamos y cuantos components no radix trabajamos



```suggestion
// radix
@import url("./radix/switch.css");
@import url("./radix/navigation-menu.css");

// custom
@import url("./components/button.css");
@import url("./components/link.css");
```